packages/rancher-prom-to-teams/generated-changes/patch/templates/configmap.yaml.patch

```diff
@@ -0,0 +1,133 @@
+--- charts-original/templates/configmap.yaml
++++ charts/templates/configmap.yaml
+@@ -8,13 +8,13 @@
+ {{- if and (not .Values.prom2teams.connector) (not .Values.prom2teams.connectors) -}}
+ {{- fail "Invalid configuration: At least one connector must be provided"}}
+ {{- end -}}
++{{- $namespace := (include "prom2teams.namespace" . ) -}}
++{{- if and (.Release.IsInstall) (not (lookup "v1" "ConfigMap" $namespace (include "prom2teams.fullname" .))) }}
+ apiVersion: v1
+ kind: ConfigMap
+ metadata:
+-  name: prom2teams-config
+-  labels:
+-    app.kubernetes.io/managed-by: {{ .Release.Service }}
+-    app.kubernetes.io/instance: {{ .Release.Name }}
++  namespace: {{ $namespace }}
++  {{- include "prom2teams.preNameLabelAnno" . | nindent 2 }}
+ data:
+   config.ini: |-
+     [HTTP Server]
+@@ -33,4 +33,110 @@
+     Level: {{ .Values.prom2teams.loglevel }}
+     [Template]
+     Path: {{ .Values.prom2teams.templatepath }}
+-  teams.j2: {{ .Files.Get "files/teams.j2" |  quote }}
+\ No newline at end of file
++  teams.j2: {{ .Files.Get "files/teams.j2" |  quote }}
++---
++apiVersion: batch/v1
```

Comment:
Instead of patching the configmap.yaml, just delete this file and call it configMap-preinstall.yaml so it's easier to review / keep up to date. Since it's fairly independent of what upstream wants out of this file.